MMA Removal Utility/README.md

```diff
@@ -192,8 +192,8 @@ The AzTS MMA Discovery and Removal utility is highly flexible and configurable.
 ### **2. What permissions does the AzTS MMA Discovery and Removal utility require?** 
```

Comment:
Here also we need to add new permissions

Reply:
Added